src/compositions/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,78 @@
+.progressBar {
+    display: grid;
+    justify-content: center;
+    padding: 3rem 0;
+}
+
+.progressBar .linesGridContainer {
+    position: absolute;
+    width: 600px;
+    left: 0;
+    right: 0;
+    margin: 111px auto 0;
+}
+
+@media only screen and (min-width: 1025px) {
```

Comment:
best to put the media queries at bottom of the file after all classes are declared